pylti/tests/test_flask.py

```diff
@@ -65,30 +64,34 @@ def setUp(self):
         self.app = app.test_client()
         app_exception.reset()
 
-    def get_exception(self):
+    @staticmethod
+    def get_exception():
         """
-        Returns exception raised by PyLTI
+        Returns exception raised by PyLTI.
         :return: exception
         """
         return app_exception.get()
 
-    def has_exception(self):
+    @staticmethod
+    def has_exception():
         """
-        Check if PyLTI raised exception
+        Check if PyLTI raised exception.
         :return: is exception raised
         """
         return app_exception.get() is not None
 
-    def get_exception_as_string(self):
+    @staticmethod
+    def get_exception_as_string():
         """
-        Return text of the exception raised by LTI
+        Return text of the exception raised by LTI.
         :return: text
         """
-        return "{}".format(self.get_exception())
+        return "{}".format(TestFlask.get_exception())
 
     def test_access_to_oauth_resource_unknown_protection(self):
+        # pylint: disable=invalid-name
```

Comment:
I tried removing all of these and there were no vioations.  Please remove all of the invalid-name disables.